RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronIDProducer.cc

```diff
@@ -124,7 +119,23 @@ void LowPtGsfElectronIDProducer::produce(edm::StreamID, edm::Event& event, const
     if (gsf.isNull()) {
       continue;
     }
-    float unbiased = (*unbiasedH)[gsf];
+
+    // Attempt to obtained ElectronSeed unbiased BDT score
+    float unbiased = -10.;
+    if (unbiasedH.isValid()) {
+      unbiased = (*unbiasedH)[gsf];
+    } else {
+      pat::Electron const* casted = dynamic_cast<pat::Electron const*>(ele.get());
+      if (casted == nullptr) {
+        std::ostringstream os;
+        os << "Problem accessing low-pT 'unbiased' ElectronSeed collection"
+           << " and a problem to dynamic_cast<pat::Electron*>(reco::GsfElectron*)!" << std::endl;
+        edm::LogError("InvalidHandle") << os.str();
```

Comment:
```suggestion
```
a LogError is pointless considering the following exception

Reply:
Removed.